sdk/python/VERSION

```diff
@@ -1,3 +0,0 @@
-Generated by pulumi-tfgen-aws from:
-Repo: github.com/terraform-providers/terraform-provider-aws
```

Comment:
Hm this edit was not expected. It's coming from adding

```
rm -rf sdk/python/
```

I wonder why this did not get regenerated, maybe this is a stale file?

Reply:
The file was generated 5 years ago by jen20.

Not present anywhere else. So this is good. This is no longer produced by the automation so is no longer accurate. We should remove it.